src/archivist/fetcher/index.test.js

```diff
@@ -17,6 +18,8 @@ chai.use(chaiAsPromised);
 
 const termsHTML = '<!DOCTYPE html><html><head><meta charset="UTF-8"><title>First provider TOS</title></head><body><h1>Terms of service</h1><p>Dapibus quis diam sagittis</p></body></html>';
 
+const fetch = args => fetchWithoutConfig({ options: config.get('fetcher'), ...args });
```

Comment:
We are not testing the exposed function anymore but a derived closure. This decreases significantly the reliability of the tests and is a smell.

Reply:
Do you suggest I should pass the `options` to each `fetch` call?

Also I'm curious if you can share an article about this being a bad practice.
Thanks!

Reply:
I suggest that we offer an API that does not us to wrap it to make testing usable 😉 if we need to change the signature to make testing it not too painful, it is very likely that reusers will also find the default signature painful.

Reply:
As for the closure decreasing the reliability, this simply has to do with the fact that we rewrite what `this` is and we `get` the config on every call. I understand that this is not how this function will be used most of the time. We are more likely to introduce (or cover) bugs this way.

Reply:
See https://github.com/ambanum/OpenTermsArchive/pull/872

Reply:
Merged in this branch